django_app/tests/test_views.py

```diff
@@ -26,6 +26,8 @@ def count_s3_objects(s3_client) -> int:
 
 @pytest.mark.django_db
 def test_upload_view(alice, client, file_pdf_path, s3_client):
+    s3_client.delete_object(Bucket=settings.BUCKET_NAME, Key=file_pdf_path.split("/")[-1].replace(" ", "_"))
```

Comment:
if the file already exist then the file-count wont increase as the file will just be overwritten